javav2/example_code/quicksight/pom.xml

```diff
@@ -36,7 +38,7 @@
             <dependency>
                 <groupId>software.amazon.awssdk</groupId>
                 <artifactId>bom</artifactId>
-                <version>2.19.14</version>
+                <version>2.20.21</version>
```

Comment:
21.20?

Reply:
That is correct --  <version>2.20.21</version>